ADOPTERS.md

```diff
@@ -0,0 +1,15 @@
+ADOPTERS
+========
+
+PUBLIC REFERENCES
+-----------------
+
+This document lists the organizations that use Quarkus based on public information available in blog posts and videos. 
+If organizations would like get added or removed please make a PR and follow these rules to comply with legal: 
+
+* Do not include the logo
+* Add a reference (link to a public blog post, video, slides, etc) mentioning that Quarkus is used
```

Comment:
What if there is no public blog post/video/etc..? I think it should be clear it's optional: we do like to have blogposts, but let's not make the barrier too high?

Reply:
I'm afraid that legal might not like having no factual/public proof about Quarkus, that's why I made it required.

Reply:
If it's people from organization X claiming they use it, there's no reason to not accept such a pull request - it's their patch after all, so their own word.

Reply:
the PR to add it or zulip chats with the claims should be good too.

Reply:
Any suggestions on how to reword that properly?